### PR TITLE
where: the `/q` parameter doesn't display anything

### PR DESCRIPTION
**Pull request by iridacea**

### CODE REVIEW

File: pages/windows/where.md [hunk 1]
```diff
@@ -16,6 +16,6 @@
 
 `where /R {{path/to/directory}} {{file_pattern}}`
 
-- Display only the error code for the location of file pattern:
+- Only return the error code for the location of file pattern:
```
Comment (kbdharun) [inline pages/windows/where.md:19]:
```suggestion
- Return the error code for the location of the file pattern:
```
I think this is better. What do you think?

Comment (iridacea) [inline pages/windows/where.md:19]:
```suggestion
- Silently return the error code for the location of the file pattern:
```
@kbdharun 
What if we change it this way to reflect the fact that nothing is displayed, and the other command also return error code?

Comment (iridacea) [reply pages/windows/where.md:19]:
```suggestion
- Silently return the error code for the location of the file pattern:
```
@kbdharun
What if we change it this way to reflect the fact that nothing is displayed, and the other command also return error code?

**Comment by iridacea** [comment]
OK, I decided to go with the second option.

**Comment by iridacea** [comment]
Looks like I accidentally removed review request for @navarroaxel  and can't return it back.

**Review by marchersimon — APPROVED** [review]
Looks good, thanks @iridacea

**Review by kbdharun — APPROVED** [review]

**Pull request merged**